libs/client-api/src/http.rs

```diff
@@ -652,6 +653,23 @@ impl Client {
       .into_data()
   }
 
+  #[instrument(level = "info", skip_all, err)]
+  pub async fn get_workspace_folder(
```

Comment:
can we add documentation for this api endpoint?

Reply:
added